### PR TITLE
Feature[TEO-141] OrganizationProfile component

### PR DESCRIPTION
**Pull request by biggicode**
https://wearetribus.atlassian.net/browse/TEO-141

### CODE REVIEW

File: frontend/src/assets/locales/ro/translation.json [hunk 1]
```diff
@@ -13,5 +13,17 @@
       "dashboard": "Dashboard"
     },
     "collapse": "Restrange meniul"
+  },
+  "organization_profile": {
```
Comment (birloiflorian) [inline frontend/src/assets/locales/ro/translation.json:17]: this should be called "organization", not necessary to be called "organization_profile".

File: frontend/src/assets/locales/ro/translation.json [hunk 2]
```diff
@@ -13,5 +13,17 @@
...
+    "title": "Profil organizație",
+    "button_label": "Editează",
```
Comment (birloiflorian) [inline frontend/src/assets/locales/ro/translation.json:19]: This should be in "general". Please align with this PR: https://github.com/code4romania/teo/pull/15/files

File: frontend/src/assets/locales/ro/translation.json [hunk 3]
```diff
@@ -13,5 +13,17 @@
...
+    "data_info": "Datele sunt preluate din ONGHub.",
```
Comment (birloiflorian) [inline frontend/src/assets/locales/ro/translation.json:20]: This is "subtitle"

File: frontend/src/assets/locales/ro/translation.json [hunk 4]
```diff
@@ -13,5 +13,17 @@
...
+    "logo": "Logo organizație",
+    "name": "Denumire organizație",
+    "email": "Email",
```
Comment (birloiflorian) [inline frontend/src/assets/locales/ro/translation.json:23]: email, phone, address are generic namings not specific to the organization and should be present in "general", please refer to the PR above.

File: frontend/src/components/OrganizationProfile.tsx [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+      <CardBody>
+        <div className="flex flex-col gap-6 w-full lg:w-[80%] mx-auto pt-4 pb-16">
+          <div className="flex flex-col gap-1">
+            <h2>{t('title')}</h2>
+            <p className="text-cool-gray-500">{t('data_info')}</p>
+          </div>
+          <div className="flex flex-col gap-1">
+            <p>{t('logo')}</p>
+            <img
+              src="logo.svg"
```
Comment (birloiflorian) [inline frontend/src/components/OrganizationProfile.tsx]: this should come from the organization.

**Review by birloiflorian — CHANGES_REQUESTED** [review]

File: frontend/src/components/OrganizationProfile.tsx [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+const OrganizationProfile = ({ organization }: OrganizationProfileProps) => {
+  const { t } = useTranslation(['organization', 'general']);
+
+  return (
+    <Card>
+      <CardHeader>
+        <h2>{t('title')}</h2>
+        <Button
+          className="btn-outline-secondary w-20"
+          label={t('edit')}
```
Comment (birloiflorian) [inline frontend/src/components/OrganizationProfile.tsx:34]: Does this translation work?

File: frontend/src/components/OrganizationProfile.tsx [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+  organization: IOrganization;
+}
+
+const OrganizationProfile = ({ organization }: OrganizationProfileProps) => {
+  const { t } = useTranslation(['organization', 'general']);
+
+  return (
+    <Card>
+      <CardHeader>
+        <h2>{t('title')}</h2>
```
Comment (birloiflorian) [inline frontend/src/components/OrganizationProfile.tsx:31]: Please validate with @OlteanuCosmin and @AndreeaMariaP for the font behaviour on mobile devices.

**Review by birloiflorian — CHANGES_REQUESTED** [review]

File: frontend/src/assets/locales/ro/translation.json [hunk 8]
```diff
@@ -10,8 +27,23 @@
...
+    "members": "Membri",
+    "branches": "Filiale si sucursale",
+    "departments": "Departamente",
+    "roles": "Roluri"
+  },
+  "organization": {
+    "title": "Profil organizație",
+    "subtitle": "Datele sunt preluate din ONGHub.",
+    "logo": "Logo organizație",
+    "organization_naming": "Denumire organizație",
```
[the remark this answers is not in the document]
Comment (birloiflorian) [reply frontend/src/assets/locales/ro/translation.json]: This is not resolved.

File: frontend/src/components/OrganizationProfile.tsx [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+      <CardBody>
+        <div className="flex flex-col gap-6 w-full lg:w-[80%] mx-auto pt-4 pb-16">
+          <div className="flex flex-col gap-1">
+            <h2>{i18n.t('organization:title')}</h2>
+            <p className="text-cool-gray-500">{i18n.t('organization:subtitle')}</p>
+          </div>
+          <div className="flex flex-col gap-1">
+            <small className="text-cool-gray-500">{i18n.t('organization:logo')}</small>
+            <img
+              src={organization.logo}
```
Comment (birloiflorian) [inline frontend/src/components/OrganizationProfile.tsx:49]: We need to handle scenario if logo is not provided. Maybe we should have a placeholder.

File: frontend/src/components/OrganizationProfile.tsx [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+
+  return (
+    <Card>
+      <CardHeader>
+        <h2>{i18n.t('organization:title')}</h2>
+        <Button
+          className="btn-outline-secondary w-20"
+          label={i18n.t('general:edit')}
+          icon={<PencilIcon className="h-5 w-5 text-cool-gray-500" />}
+          onClick={() => navigate('edit')}
```
[the remark this answers is not in the document]
Comment (biggicode) [reply frontend/src/components/OrganizationProfile.tsx]: Done

File: frontend/src/pages/Organization.tsx [hunk 11]
```diff
@@ -14,6 +16,8 @@ const Organization = () => {
   const [orderByColumn, setOrderByColumn] = useState<string>();
   const [orderDirection, setOrderDirection] = useState<OrderDirection>();
 
+  const { data: organization, error: organizationProfileError } = useOrganizationProfileQuery();
```
[the remark this answers is not in the document]
Comment (biggicode) [reply frontend/src/pages/Organization.tsx]: Done

**Pull request merged**